.github/BUILD.md

```diff
@@ -13,81 +13,109 @@ If the answer is yes, please read on. Otherwise, make sure to head on over [to t
-npm i
-npm run build:for:cms
+cd <solution root>\src\Umbraco.Web.UI
+dotnet run --no-build
 ```
 
-If you want to make changes to the UI, you can choose to run a front-end development server. To learn more please read the Umbraco.Web.UI.Client README.md [Run against a local Umbraco instance](../src/Umbraco.Web.UI.Client/.github/README.md#run-against-a-local-umbraco-instance) for more information.
+When the page loads in your web browser, you can follow the installer to set up a database for debugging. When complete, you will have an empty Umbraco installation to begin working with. You may also wish to install a [starter kit][https://marketplace.umbraco.com/category/themes-&-starter-kits] to ease your debugging.
 
-The login screen is a different frontend build, for that one you can run it as follows:
+### Back-end only changes
+
+If you are working on back-end only features, when switching branches or pulling down the latest from GitHub, you will find the front-end getting rebuilt periodically when you look to build the back-end changes. This can take a while and slow you down. So if for a period of time you don't care about changes in the front-end, you can disable this build step.
+
+Go to `Umbraco.Cms.StaticAssets.csproj` and comment out the following lines of MsBuild by adding a REM statement in front:
 
 ```
-cd src\Umbraco.Web.UI.Login
-npm i
-npm run dev
+REM npm ci --no-fund --no-audit --prefer-offline
+REM npm run build:for:cms
 ```
 
-If you just want to build the Login screen to `Umbraco.Web.UI` then instead of running `dev`, you can do: `npm run build`.
+Just be careful not to include this change in your PR.
+
+### Front-end only changes
+
+Conversely, if you are working on front-end only, you want to build the back-end once and then run it. Before you do so, update the configuration in `appSettings.json` to add the following under `Umbraco:Cms:Security`:
+
+```
+"BackOfficeHost": "http://localhost:5173",
+"AuthorizeCallbackPathName": "/oauth_complete",
+"AuthorizeCallbackLogoutPathName": "/logout",
+"AuthorizeCallbackErrorPathName": "/error"
+```
 
-To run the C# portion of the project, either hit <kbd>F5</kbd> to begin debugging, or manually using the command line:
+Then run Umbraco from the command line.
 
 ```
-dotnet watch --project .\src\Umbraco.Web.UI\Umbraco.Web.UI.csproj
+cd <solution root>\src\Umbraco.Web.UI
+dotnet run --no-build
 ```
 
-**The initial C# build might take a _really_ long time (seriously, go and make a cup of coffee!) - but don't worry, this will be faster on subsequent runs.**
+In another terminal window, run the following to watch the front-end changes and launch Umbraco using the URL indicated from this task.
 
-When the page eventually loads in your web browser, you can follow the installer to set up a database for debugging. You may also wish to install a [starter kit][starter kits] to ease your debugging.
+```
+cd <solution root>\src\Umbraco.Web.UI.Client
+npm run dev:server
```

Comment:
do an `npm ci` first if it's never had the package restore process run before?

Reply:
I've added a single reference to when you might have to do this and how you should do it.